crawler.js

```diff
@@ -36,7 +36,8 @@ function openBrowser(log, proxyHost, executablePath) {
         args: [
             // enable FLoC
             '--enable-blink-features=InterestCohortAPI',
-            '--enable-features="FederatedLearningOfCohorts:update_interval/10s/minimum_history_domain_size_required/1,FlocIdSortingLshBasedComputation,InterestCohortFeaturePolicy"'
+            '--enable-features="FederatedLearningOfCohorts:update_interval/10s/minimum_history_domain_size_required/1,FlocIdSortingLshBasedComputation,InterestCohortFeaturePolicy"',
+            '--js-flags="--async-stack-traces --stack-trace-limit 32"'
```

Comment:
With this change error breakpoints now include async traces. Why we decided to keep Debugger.pause too? I remember we discussed it but forgot the summary.

Reply:
- first, I think async stack traces are actually enabled by default in Chrome, this is just to make sure
- The main reason to keep Debugger.pause is that runtime stacks still don't include real URLs in case of sourcemaps. So with things like "webpack-internal://" we have more chances since we can read `embedderName` in CDP